ormin/ormin_postgre.nim

```diff
@@ -11,6 +11,7 @@ type
   varchar* = string
   integer* = int
   timestamp* = string
+  uuid* = string
```

Comment:
Unused.

Reply:
Very much used: [Postgres UUID Type](https://www.postgresql.org/docs/11/datatype-uuid.html)
Without it the following error message appears: 
`/ormin/ormin/queries.nim(141, 17) Error: undeclared identifier: 'uuid'`

Reply:
ok.